sdcm/tester.py

```diff
@@ -958,11 +959,7 @@ def setUp(self):  # pylint: disable=too-many-branches,too-many-statements
             self.run_commit_log_check_thread(self.get_duration(None))
 
     def set_system_auth_rf(self, db_cluster=None):
-
-        if not db_cluster:
-            db_cluster = self.db_cluster
-        # No need to change system tables when running via scylla-cloud
-        # Also, when running a Alternator via scylla-cloud, we don't have CQL access to the cluster
+        db_cluster = db_cluster or self.db_cluster
```

Comment:
It looks redundant, because the `db_cluster` discovery gets handled by the `set_ks_rf` method.

Reply:
Yes, but you need it for the `if len(db_cluster.nodes) == 1` check

Reply:
Ok.